docs/AssetAttributes.md

```diff
@@ -50,7 +50,7 @@ Method: POST
 | assetAttribute.description | <code>string</code> |  |
 | [assetAttribute.isRequired] | <code>boolean</code> |  |
 | assetAttribute.label | <code>string</code> |  |
-| assetAttribute.organizationId | <code>string</code> |  |
+| assetAttribute.organizationId | <code>string</code> | Can be explicitly set to `null` to create a global attribute  |
```

Comment:
These are automatically generated so it's best not to make the change here and instead make it in the JSDoc stuff in `src/assets/assetAttributes.js` above the `create` method [here](https://github.com/ndustrialio/contxt-sdk-js/blob/master/src/assets/assetAttributes.js#L99)